shared/actions/chat/index.js

```diff
@@ -318,7 +318,7 @@ function * _loadMoreMessages (action: Constants.LoadMoreMessages): SagaGenerator
     let newMessages = []
     messages.forEach((message, idx) => {
       if (idx > 0) {
-        const timestamp = Shared.maybeAddTimestamp(messages[idx], messages, idx - 1)
+        const timestamp = Shared.maybeAddTimestamp(messages[idx], List(messages), idx - 1)
```

Comment:
Instead of converting here, you can use `Immutable.Seq` to get a `findLast` that works on Arrays, Lists, etc.